SPECS/xs-openssl.spec

```diff
@@ -552,10 +552,11 @@ export LD_LIBRARY_PATH
 %postun libs -p /sbin/ldconfig
 
 %changelog
-* Fri Mar 31 2024 David Morel <david.morel@vates.tech> - 1:1.1.1k-12.1
+* Fri Mar 31 2024 David Morel <david.morel@vates.tech> - 1:1.1.1k-12.2
```

Comment:
We don't reuse a changelog entry of a release which was already sent as a regular build to Koji. It's already consumed. You need to create a separate entry now.

Reply:
The version we already built is the .2 one, I had consumed it by mistake and the idea was to keep it as we discussed previously. As I somehow missed this patch when merging the previous version, it stayed on .1 in git, but the package we have and that went through CI is already the .2 with this patch.

I can either:
- keep it as is
- make a separate changelog for clarity
- directly move to .3 and go through a rebuild

Let me know what you prefer, no problem for me with any of the solutions.

Reply:
If indeed the 12.2 that was built had the change, then it's good as you did it.